src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -345,13 +345,15 @@ These tasks can be easily integrated to your AI pipelines/workflows as you can s
 
 The management of the life cycle of ML MaaS will be detailed in the next sub-sections.
 
-Once an ML MaaS instance is up and running, it could be used for:
+Once a ML MaaS instance is up and running, it could be used for:
 
-- *AI Model Deployment or Update*: the user has to provide a valid identifier in order to deploy the model of his/her choice.
+- *AI Model Deployment or Update*: the user has to provide a valid specific AI Model identifier in order to deploy the model of his/her choice.
 
-- *Call of Predictions*: the user requests to give predictions for a specific payload that has to be converted into json data in order to get prediction's values.
+- *Call of Predictions*: when a specific AI model is running, the user can request predictions for a specific payload. This latter has to be converted into json data in order to get prediction's values.
 
-Using ML MaaS, you can easily deploy and use any machine learning model as a Web Service on a physical or a virtual compute host on which there is an available Proactive Node. Going through the Proactive Scheduler,
+- *Deploy a New Specific AI Model*: the running generic AI model can be used to deploy a new specific AI model.
+
+Using ML MaaS, you can easily deploy and use any machine learning model as a REST Web Service on a physical or a virtual compute host on which there is an available Proactive Node. Going through the Proactive Scheduler,
```

Comment:
on a physical or a virtual compute host => on a physical or virtual compute host

Reply:
This is not a mistake, I prefer to let what Imen wrote.